### DISCUSSION TITLE
Possibilité d'entendre toutes les voix des radios qu'on a

### DISCUSSION THREAD

**Discussion by ph1823** [ZeEvoker2Vex/Z-Radio-Addon#1] [Ideas]
Bonjours 👋
Une petite suggestion pour les serveurs RP qui serait configurable.

Comme dans la vrai, on peut posséder plusieurs radio (c'est rare mais ça existe), et in entend toutes les voix de toutes les radio, la suggestion est donc simple pouvoir entendre toutes les radios qu'on possède sur nous (c'est peut être compliqué à mettre en place 😅)

**Comment by SeedenDev** [comment]
Bonjour, et désolé pour le retard à cette réponse ^^

J'approuve totalement cette suggestion d'un côté, j'y avais pensé mais pour simplifier au début je n'ai pas choisi d'intégrer un tel système qui, aussi pourrait vite devenir le bazar à celui qui en posséderait plusieurs, mais après tout ^^

Je mettrais donc ce système je pense, en option, sûrement dans la config du serveur. Les joueurs auront qu'à bien se tenir et à éteindre leur radio ^^

Merci pour votre suggestion, comme une autre précédent proposée sur Discord, je l'appliquerais dans une MAJ après la sortie de mon autre addon.

Cordialement, ZeEvoker2Vex.

**Comment by SeedenDev** [comment]
> Bonjours 👋
> Une petite suggestion pour les serveurs RP qui serait configurable.
> 
> Comme dans la vrai, on peut posséder plusieurs radio (c'est rare mais ça existe), et in entend toutes les voix de toutes les radio, la suggestion est donc simple pouvoir entendre toutes les radios qu'on possède sur nous (c'est peut être compliqué à mettre en place 😅)

Bonsoir !
Je vous notifie pour vous signaler que cette fonctionnalité a été développée les jours précédents et que la MAJ 1.1 devrait sortir d'ici quelques jours, comme il reste l'une des nouveautés principales à faire.

Tenez-vous au courant sur le Discord de ModularVoiceChat dans le salon #z-radio.

En vous souhaitant une bonne fin de soirée, ZeEvoker2Vex.

**Comment by SeedenDev** [comment]
> Bonjours 👋 Une petite suggestion pour les serveurs RP qui serait configurable.
> 
> Comme dans la vrai, on peut posséder plusieurs radio (c'est rare mais ça existe), et in entend toutes les voix de toutes les radio, la suggestion est donc simple pouvoir entendre toutes les radios qu'on possède sur nous (c'est peut être compliqué à mettre en place 😅)

Bonsoir, je vous annonce que la mise à jour contenant votre suggestion est sortie.

Bonne soirée, ZeEvoker2Vex.
